### PR TITLE
Disable retpoline on Windows by default until fixed

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Disable retpoline code on Windows until #320 is fixed.

### CODE REVIEW

File: cmake/settings.cmake [hunk 1]
```diff
@@ -26,13 +26,7 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
       CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
 
       target_compile_options("ubpf_settings" INTERFACE
-        -g
```
Comment (hawkinsw) [inline cmake/settings.cmake:29]: MARK (see below)

File: vm/ubpf_config.h.inc [hunk 2]
```diff
@@ -8,5 +8,6 @@
 
 #pragma once
 
+#cmakedefine UBPF_DISABLE_RETPOLINES
```
Comment (hawkinsw) [inline vm/ubpf_config.h.inc:11]: It seems like *this* is the recommended way to do what I did about (at MARK) manually? If so, that's awesome! I learned something new!

**Review by hawkinsw — COMMENTED** [review]
Just a quick Q.

File: vm/ubpf_config.h.inc (hunk 2 above)
Comment (Alan-Jowett) [reply vm/ubpf_config.h.inc:11]: Not sure to be honest, but this approach works with both CMake and Visual Studio builds.

Comment (hawkinsw) [reply vm/ubpf_config.h.inc:11]: Great!

**Review by hawkinsw — APPROVED** [review]

**Pull request merged**